picky/src/key/ec.rs

```diff
@@ -65,13 +65,13 @@ impl<'a> TryFrom<&'a PrivateKey> for EcdsaKeypair<'a> {
 
         Ok(EcdsaKeypair {
             private_key: private_key_data,
-            public_key: public_key_data,
+            _public_key: public_key_data,
             curve,
         })
     }
 }
 pub(crate) struct EcdsaPublicKey<'a> {
-    pub(crate) data: &'a [u8],
+    pub(crate) _data: &'a [u8],
```

Comment:
I got confused by the `_` prefix, I thought this `_data` attribute was no longer used. Shouldn't we leave the name unchanged?

Reply:
Yes, I'll fix this. the `_` was left during coding by mistake